hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/clients/DefaultClientRegistry.java

```diff
@@ -84,6 +84,11 @@ public DefaultClientRegistry(Config config, Executor cacheLoaderExecutor) {
             .build();
   }
 
+  @Override
+  public GrpcChannelRegistry getChannelRegistry() {
```

Comment:
Looks good to me, can we keep these changes when we work on trace-enricher changes?